pkg/search/postgres/query/array_query.go

```diff
@@ -22,15 +21,15 @@ func getStringArrayPostTransformFunc(entry *QueryEntry) (func(val interface{}) i
 		return nil, errors.New("no filter func found")
 	}
 	return func(val interface{}) interface{} {
-		textArray, _ := val.(*pgtype.TextArray)
+		textArray, _ := val.(*[]string)
```

Comment:
Looking at the pgx changelog, it seems that arrays are now generic types. Did you consider using https://github.com/jackc/pgx/blob/038fc448c1cc91b9372a6a0d151789f41536f033/pgtype/array.go#L379 instead of `[]string`?

Reply:
Good point. Since the `TestSearchComparison/TestDeploymentSearchResults/test#02` was failing and when I attached debugger to it shows me that `val` is `[]string` so I did not try `pgtype.Array` until now and it failed as conversion will not work.